_posts/blog/2017-05-15-how-to-structure-product-feedback.md

```diff
@@ -0,0 +1,42 @@
+
+Statements like this imply other people share your experience (which is unclear without supporting data) and can easily lead to misinformed product decisions. Before you start giving feedback, make a conscious effort to think about whether you are speaking for **yourself** or for other people. 
+
+## Provide Context
+Providing context to your experience gives listeners insight into your goals and motivations and will help them understand where your feedback is coming from. Describe **what** you were trying to accomplish.
+
+> “I was trying to purchase a bag of coffee.”
+
+## Describe What You Expected
+We all have expectations (mental models) of how products should look and work, based on previous experiences. Describing what you expected is a good way to frame your feedback — by clarifying your expected output or outcome, you’re also empowering the listener to more effectively empathize with your point of view. Speak about **your** experience and describe what **you** expected to see or happen.
+
+> “I expected to see the buy button at the top of the page, because that’s where it is on a lot of the sites I use.”
+
+## Describe What Actually Happened
+What you expect to happen and what actually happens sometimes don’t align. Drawing this comparison highlights how design and implementation impact the experience. After describing what you expected to happen, describe what **actually** happened. 
+
+>"I was never able to find the button to complete my purchase."
+
+Now the person receiving your feedback will have a clear understanding what you were trying to accomplish, what you expected to happen, and what actually happened. 
+
+## Suggestions 
+If you want to productively give a suggestion on how to improve your experience, focus on the goal, not the solution to the goal. Leave the implementation of the  goal up to the person or team receiving the feedback. This will show the team you have confidence in their ability to address your problems and improve your experience.
+
+> “My experience would be better if it were easier to buy the coffee.”  
+
+## Have Some Chill
+If I get emotional when providing feedback, it really alienates the person I’m speaking with and makes the experience I’m trying to convey even harder to understand. We are all on the same team and share the same goals. So, before you get all hot and bothered when providing feedback, pause for a moment and calm yourself.
+
+## Putting It All Together
+If you use these steps to structure your thoughts, you will end up with objective, precise, and informative feedback. 
+
+>“I was trying to purchase a bag of coffee and I expected to see the buy button at the top of the page, because that’s where it is on a lot of the sites I use. I was never able to find the button, so I wasn’t able to complete my purchase. I would have had a better experience if I were able to buy the coffee.”
+
+Thanks for checking out this post. I hope you found it useful.
+
+
```

Comment:
No need for empty lines at the end of the post, you should be able to take these out.